src/sb_pointcloud_processing/src/LineExtractorNode.cpp

```diff
@@ -176,7 +176,7 @@ std::vector<mapping_igvc::LineObstacle> line_obstacles) {
     snowbots::RvizUtils::createrMarkerScale(
     this->scale, this->scale, this->scale);
 
-    std::string frame_id = "line_extractor_test";
+    std::string frame_id = "base_link";
```

Comment:
There's a merge conflict here; we should use the changes from #295 (already merged into master), which allows `frame_id` to be configurable from launch file.

Reply:
👌